docs/controls/WebView-known-issues.md

```diff
@@ -0,0 +1,112 @@
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, WebView, Windows Forms, WPF, known issues, release notes
+---
+
+# Known Issues of the WebView control for Windows Forms and WPF applications
+
+This document describes issues and limitations in this release of the **WebView** control.
+
+## WebView control members
+
+The **WebView** control does not implement these methods of the [IWebViewControl](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol).
+
+* [NavigateToLocalStreamUri](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.navigatetolocalstreamuri)
+
+* [NavigateWithHttpRequestMessage](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.navigatewithhttprequestmessage)
+
+* [CapturePreviewToStreamAsync](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.capturepreviewtostreamasync)
+
+* [CaptureSelectedContentToDataPackageAsync](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.captureselectedcontenttodatapackageasync)
+
+* [BuildLocalStreamUri](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.buildlocalstreamuri)
+
+The **WebView** control implements these events of the [IWebViewControl](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol), but they don't work as expected.
+
+* [LongRunningScriptDetected](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.longrunningscriptdetected)
+
+* [UnsafeContentWarningDisplaying](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.unsafecontentwarningdisplaying)
+
+* [UnsupportedUriSchemeIdentified](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.unsupportedurischemeidentified)
+
+* [UnviewableContentIdentified](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.unviewablecontentidentified)
+
+* [NewWindowRequested](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.newwindowrequested)
+
+## Rendering and layout
+
+* The **WebView** control doesn't appear at the correct scale when users move the host application between monitors that have different screen resolutions.
+
+* The **WebView** control doesn't appear as expected on some high-resolution displays.
+
+* The [Control.Bounds](https://msdn.microsoft.com/library/system.windows.forms.control.bounds.aspx) property is not supported.
+
+## Performance
+
+* **WebView** controls in WPF applications have longer load times than Windows Forms applications.
+
+* If the web view hangs or stops working, other applications that use the same underlying process (Win32WebViewHost) will hang or stop working as well.
+
+## Security
+
+* **WebView** controls don't render in XAML Browser Applications (XBAP).
+
+  That's because a browser has partial trust to the system while WebView controls have full-trust.
+
+* **WebView** controls won't appear if the Win32WebViewHost application is disabled in the location settings of the user's system.
```

Comment:
The WebView appears; however the permission being requested is denied despite the user's response in the application to allow. This is true for other permissions as well (e.g. media)